main.go

```diff
@@ -19,21 +19,17 @@ package main
 
 import (
 	"flag"
-	"io"
 	"log"
-	"net"
 	"net/url"
 	"strings"
 
-	"github.com/inconshreveable/go-vhost"
 	"github.com/mysteriumnetwork/openvpn-forwarder/api"
 	"github.com/mysteriumnetwork/openvpn-forwarder/proxy"
 	netproxy "golang.org/x/net/proxy"
 )
 
+var proxyAddr = flag.String("proxy.bind", ":8080", "Proxy address for incoming connections")
```

Comment:
Port 8443, because it's HTTPS proxy - we connect to final endpoint) without proxying request/response itself

Reply:
Changed.